src/lib/connectors/db.js

```diff
@@ -13,21 +13,7 @@ pool.on('acquire', () => {
 });
 
 function promiseQuery (queryString, params) {
```

Comment:
Just wondering if the shape of the response has changed here

Reply:
Good spot. Updated that now thanks.